doc/topics/releases/3005.rst

```diff
@@ -1,7 +1,135 @@
 .. _release-3005:
 
 =============================================
-Salt 3005 Release Notes - Codename Phosphorus
+Salt 3005 release notes - Codename Phosphorus
 =============================================
 
 Salt 3005 is currently under development.
+
+
+Python 3.5 and 3.6 Deprecation
+------------------------------
+
+This will be the last release we will support Python versions 3.5 and 3.6.
+In Salt release 3006, we will only support Python versions 3.7 and higher.
+Going forward, our policy will be to align with Python's supported versions.
+
+
+New packages available
+----------------------
+
+With the release of Salt 3005, we are pleased to announce the
+new packages using Tiamat and pyinstaller are now out of beta
+and ready for production. These new packages are self-contained
+binaries of Salt, which includes the version of Python needed by Salt
+and the required dependencies of Salt. The Tiamat packages allow users
+to use Salt without Python being installed on the box.
+
+Any new OS platforms we support going forward from version 3005 will only
+include Tiamat packages. For this release, this includes Centos 8 Stream,
+Centos 9 Stream and Ubuntu 22.04.
+
+On the day of the Phosphorus release, the instructions on how to install the
+new Tiamat packages will be available on https://repo.saltproject.io for each
```

Comment:
**Chore (future action needed, not sure if this is blocking):** The instructions won't be on repo, they'll be on the new Salt install guide. See my previous comment about the URL. Do you need me to expedite the URL request because it will block these release notes?